Test/DependencyCollector/FunctionalTests/TestApps/Aspx451/ApplicationInsights.config

```diff
@@ -47,26 +47,6 @@
         <Add>127.0.0.1</Add>
       </ExcludeComponentCorrelationHttpHeadersOnDomains>
     </Add>
-    <Add Type="Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.PerformanceCollectorModule, Microsoft.AI.PerfCounterCollector">
```

Comment:
Removed intentionally?

Reply:
there were no reference on performance counter project anyway. So I figured no need to throw extra exceptions on loading